### DISCUSSION TITLE
Elements other than <button> for custom event goals?

### DISCUSSION THREAD

**Discussion by leonvh4** [plausible/analytics#3259] [Anything Else]
Hi everyone. I am looking to track clicks on specific buttons on my site. The site is built on Webflow, so we cannot use the `<button>` tag, it is an `<a>` tag as it links to a secondary site. Plausible is picking these clicks up as outbound clicks, and the `Outbound Link: Click` goals are working fine. But, I would like to track each button, as some of them link to the same secondary site.

What is the best element to use to capture these clicks as custom events? 

All the steps as outlined here have been followed: https://plausible.io/docs/custom-event-goals

Thanks!

**Comment by metmarkosaric** [comment]
Hi @leonvh4! We have a Webflow guide here https://plausible.io/docs/webflow-integration. Perhaps that's useful?

**Reply by leonvh4** [reply]
@metmarkosaric That will be very useful, thanks a lot!
